gladier/managers/flows_manager.py

```diff
@@ -252,9 +252,14 @@ def get_flow_permission(self, permission_type, identities=None):
         return identities
 
     def get_flow_id(self) -> str:
-        """Return flow id. If an ID was set on this class in the constructor, that is
-        used. Otherwise, a retrieve from storage is attempted with 'flow_id'.
-        :returns: flow_id uuid for deployed flow, or None if it does not exist"""
+        """
+        Return flow id. If an ID was set on this class in the constructor, that is
+        used. Otherwise, a retrieve from storage is attempted with 'flow_id' if a
+        flow has been run before and a local id is cached in storage. If no flow_id
+        exists from either of these locations, None is returned instead.
+
+        :returns: flow_id uuid for deployed flow, or None if it does not exist
```

Comment:
The type annotation on the return is just `str`. According to this doc, it should probably be `Optional[str]`

Reply:
This is fixed in the latest commit, although the changes were a little contagious in fixing consistency in the manager class. They were small though.

Thanks, and let me know if you see anything else!